### ISSUE TITLE
`jsx-a11y/aria-proptypes` errors: "The prop value with an expression type of JSXElement could not be resolved"

### ISSUE THREAD

**Issue by ljharb** [evcohen/eslint-plugin-jsx-a11y#63]
When I enable the `jsx-a11y/aria-proptypes` rule, I get the following error:
```sh
The prop value with an expression type of JSXElement could not be resolved.
  Please file issue to get this fixed immediately.
Error: The prop value with an expression type of JSXElement could not be resolved.
  Please file issue to get this fixed immediately.
    at Object.extractLiteral [as JSXExpressionContainer] ($PWD/node_modules/eslint-plugin-jsx-a11y/node_modules/jsx-ast-utils/lib/values/expressions/index.js:162:11)
    at getLiteralValue ($PWD/node_modules/eslint-plugin-jsx-a11y/node_modules/jsx-ast-utils/lib/values/index.js:65:35)
    at extractValue ($PWD/node_modules/eslint-plugin-jsx-a11y/node_modules/jsx-ast-utils/lib/getPropValue.js:24:12)
    at getLiteralPropValue ($PWD/node_modules/eslint-plugin-jsx-a11y/node_modules/jsx-ast-utils/lib/getPropValue.js:56:10)
    at EventEmitter.JSXAttribute ($PWD/node_modules/eslint-plugin-jsx-a11y/lib/rules/aria-proptypes.js:71:56)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.enterNode ($PWD/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode ($PWD/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:607:23)
    at CommentEventGenerator.enterNode ($PWD/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.traverser.traverse.enter ($PWD/node_modules/eslint/lib/eslint.js:905:36)
```

I have no idea how to find out on which file it's erroring out, but it seems like it might be a bug in `jsx-ast-utils`.

**Comment by ljharb** [comment]
Any update on this one?

**Comment by evcohen** [comment]
@ljharb will work on this today.

**Comment by evcohen** [comment]
Fixed in [jsx-ast-utils v1.3.1](https://github.com/evcohen/jsx-ast-utils/releases/tag/1.3.1) you can reinstall and should be good.
